fe/fe-core/src/main/java/org/apache/doris/analysis/AggregateInfo.java

```diff
@@ -160,6 +161,15 @@ public void setPartitionExprs(List<Expr> exprs) {
         partitionExprs = exprs;
     }
 
+    private static void validateGroupingExprs(List<Expr> groupingExprs) throws AnalysisException {
```

Comment:
I think it is more reasonable to put this checking logic into `GroupByClause`'s analyze phase.

Reply:
Thanks for your time. But there is no group by clause in sql like "select distinct bitmap, hll from xxx", the agg node is created by fe to implement the "distinct" operator, so no analyze function is called in that case. Anyway, I update the error message which is consistent with the error "select count(*) from xxx group by bitmap" now.